.travis.yml

```diff
@@ -1,61 +1,23 @@
-# ----------------------------------------
-# Python support matrix per Django version
-#
-#   2.6   1.5  1.6
-#   2.7   1.5  1.6  1.7  1.8  1.9  1.10
-#   3.3   1.5  1.6  1.7  1.8
-#   3.4             1.7  1.8  1.9  1.10
-#   3.5                  1.8  1.9  1.10
-# ----------------------------------------
 sudo: false
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "pypy"
-  - "pypy3"
-env:
-  - DJANGO='Django<1.6'  # Django 1.5
-  - DJANGO='Django<1.7'  # Django 1.6
-  - DJANGO='Django<1.8'  # Django 1.7
-  - DJANGO='Django<1.9'  # Django 1.8
-  - DJANGO='Django<1.10' # Django 1.9
-  - DJANGO='Django<1.11' # Django 1.10
 matrix:
-  exclude:
-    - python: "2.6"
-      env: DJANGO='Django<1.8'
-    - python: "2.6"
-      env: DJANGO='Django<1.9'
-    - python: "2.6"
-      env: DJANGO='Django<1.10'
-    - python: "2.6"
-      env: DJANGO='Django<1.11'
-    - python: "3.3"
-      env: DJANGO='Django<1.10'
-    - python: "3.3"
-      env: DJANGO='Django<1.11'
-    - python: "3.4"
-      env: DJANGO='Django<1.6'
-    - python: "3.4"
-      env: DJANGO='Django<1.7'
-    - python: "3.5"
-      env: DJANGO='Django<1.6'
-    - python: "3.5"
-      env: DJANGO='Django<1.7'
-    - python: "3.5"
-      env: DJANGO='Django<1.8'
-    - python: "pypy3"
-      env: DJANGO='Django<1.10'
-    - python: "pypy3"
-      env: DJANGO='Django<1.11'
+  include:
+    - env: TOXENV=py27-django111
```

Comment:
btw, I recently learned this syntax can be cleaned up like so:

```yml
matrix:
  include:
    - { python: 2.7, env: TOXENV=py27-django111 }
    - { python: 3.4, env: TOXENV=py34-django111 }
    - { python: 3.5, env: TOXENV=py35-django111 }
    - { python: 3.6, env: TOXENV=py36-django111 }
    ...
```

ref: encode/django-rest-framework/pull/5519

Reply:
thx @rpkilby for the suggestion.

@dfunckt what do you think ?

Reply:
@ticosax @rpkilby yeah, that is readable, let's do it 👍

Reply:
alright, done.